tbx/core/models.py

```diff
@@ -17,7 +16,6 @@
 from wagtail.models import Orderable, Page
 from wagtail.snippets.models import register_snippet
 
-from .api import PeopleHRFeed
```

Comment:
we could potentially remove the PeopleHRFeed code in the api as well.

Reply:
Thanks, removed in 5c0e485